src/include/executor/executor.h

```diff
@@ -87,6 +87,9 @@ extern PGDLLIMPORT ExecutorCheckPerms_hook_type ExecutorCheckPerms_hook;
 typedef bool (*TriggerRecuresiveCheck_hook_type) (ResultRelInfo *resultRelInfo);
 extern PGDLLIMPORT TriggerRecuresiveCheck_hook_type TriggerRecuresiveCheck_hook;
 
+typedef bool (*ViewInsteadStmtTrigger_hook_type) (Relation view, CmdType event);
```

Comment:
Can we have a better naming for this hook ?

Reply:
Renamed bbfViewHasInsteadofTrigger_hook_type